src/fs/rock/RockRebuild.h

```diff
@@ -126,6 +126,9 @@ class Rebuild: public AsyncJob, private IndependentRunner
     const bool resuming;
 
     static void Steps(void *data);
+
+    /// informs Coordinator that we are still foreground-rebuilding
+    void extendWait();
```

Comment:
Please move this declaration, _above_ data members, between sameEntry() and progressDescription(). Please move this _private_ method description to the method definition.